styles/angular-patternfly.css

```diff
@@ -413,3 +413,11 @@
   border-color: #0076b7 !important;
   color: #fff !important;
 }
+
+.toolbar-pf-actions .toolbar-pf-view-selector a {
+  cursor: pointer;
+}
+
+.toolbar-pf-actions .dropdown-menu a {
+  cursor: pointer;
```

Comment:
Thoughts about making an `<a>` always have `cursor: pointer;` not just within the toolbar?